tests/dash/dash_db.py

```diff
@@ -131,6 +137,19 @@ def polling_function():
         _, keys = wait_for_result(polling_function, failure_message=f"Found fewer than {min_keys} keys in ASIC_DB table {table_name}")
         return keys
 
+    def wait_for_asic_db_num_keys(self, table_name, num_expected):
```

Comment:
Don't think we need an entirely new function for this, suggest to add a `strict` option to `wait_for_db_keys` to use check for strictly == instead of >=